### PR TITLE
Set PA date time types for publish/embargo input.

### PR DESCRIPTION
**Pull request by naomiaro**

### CODE REVIEW

File: client/app/index.html [hunk 1]
```diff
@@ -246,6 +246,12 @@
 
         config.previewSubjectFilterKey = 'patopic';
 
+        //date formats for publish/embargo entry
+        config.view = {
+            dateformat: 'D/M/YYYY',
+            timeformat: 'H:mm'
```
Comment (actionless) [inline client/app/index.html]: `HH:mm`?

File: client/app/index.html [hunk 2]
```diff
@@ -246,6 +246,12 @@
 
         config.previewSubjectFilterKey = 'patopic';
 
+        //date formats for publish/embargo entry
+        config.view = {
+            dateformat: 'D/M/YYYY',
```
Comment (actionless) [inline client/app/index.html]: it was `MM/DD/YYYY` by default before

Comment (naomiaro) [inline client/app/index.html]: yeah, this time format is for input only in the angular view, the angular model reformats it to server format. The singular just allows them to type 4/4/2016 to save time 9:00 etc...

Comment (naomiaro) [inline client/app/index.html]: It's moment js format strings.

Comment (naomiaro) [inline client/app/index.html]: The model formats are

`model: {
       dateformat: 'DD/MM/YYYY',
       timeformat: 'HH:mm:ss'
},`

**Comment by mugurrus** [comment]
:+1:

**Pull request merged**